src/interpret/lianapy/config.vsh.yaml

```diff
@@ -0,0 +1,84 @@
+functionality:
+  name: lianapy
+  namespace: "interpret"
```

Comment:
Sorry I overlooked this, but could you add yourself as author? :)

Reply:
Done